### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by chaoer** [baaivision/See3D#20]
Congratulations.

And, when will you release the rest of the code for this amazing work? (trainning scripts、data curation pipeline、 and balabala)

**Comment by fangchuan** [comment]
+1. Great Work! Looking forward to the training code.

Additionally, I feel confused about the details of Time-dependent `Visual COndition`. I donot know if my understanding is correct, I hope you can help me figure out the key insight of the concept of Visual Condition. 
The `C_t` in the Sec.3.2.2 seems like a different noisy target (radom masked)image latent, the time-dependent mixture of `C_t` and `X_t` is aim to keep some visual information of target views at the larger timesteps (e.g T=999) , right? If so, I am suspicious about the design, bc mostly, we apply stronger noise at larger timesteps to destory the multi-view information, to better learn the multi-view distribution, techs like zero-snr, edm noise-scheduler and so on. Why **See3D** chosed to keep target view information? it's wired.  @mabaorui @GaoHchen
